packages/flutter_tools/test/general.shard/compile_incremental_test.dart

```diff
@@ -187,16 +197,18 @@ Future<void> _recompile(
     streamController.add(utf8.encode(mockCompilerOutput));
   });
   final CompilerOutput output = await generator.recompile(
-    null /* mainPath */,
+    Uri.parse('/path/to/main.dart'),
     <Uri>[Uri.parse('/path/to/main.dart')],
     outputPath: '/build/',
+    packageConfig: PackageConfig.empty,
   );
   expect(output.outputFilename, equals('/path/to/main.dart.dill'));
   final String commands = mockFrontendServerStdIn.getAndClear();
-  final RegExp re = RegExp(r'^recompile (.*)\n/path/to/main.dart\n(.*)\n$');
-  expect(commands, matches(re));
-  final Match match = re.firstMatch(commands);
-  expect(match[1] == match[2], isTrue);
+  final RegExp whitespace = RegExp(r'\s+');
+  final List<String> parts = commands.split(whitespace);
+
+  // Test that uuid matches at beginning and end.
+  expect(parts[2], equals(parts[4]));
```

Comment:
Previously the expected format was more or less documented in the regex. Maybe shift the expected format to a comment?

Reply:
Done